processing/data_collection/gazette/spiders/rn_mossoro.py

```diff
@@ -0,0 +1,56 @@
+from dateparser import parse
+import datetime as dt
+
+import scrapy
+from scrapy.http import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RnMossoroSpider(BaseGazetteSpider):
+    MUNICIPALITY_ID = '2408003'
+    PDF_URL = 'https://www.prefeiturademossoro.com.br/jom/{}'
+
+    GAZETTES_CSS = 'body > div > table:first-child > tr > td > table:last-child > tr > td > div > table > tr > td'
+    PDF_INFO_CSS = 'td.TextoLink'
+    PDF_REALTIVE_PATH_CSS = 'a::attr(href)'
+    DATE_REGEX = r'[0-9]{1,2} de [A-Za-z].* de [0-9]{4}'
+    IS_EXTRA_REGEX = r'-[A-Za-z]'
+
+    allowed_domains = ['prefeiturademossoro.com.br']
+    name = 'rn_mossoro'
+    start_urls = ['https://www.prefeiturademossoro.com.br/jom/alljoms.php']
+
+    def parse(self, response):
+        dates = set()
+        for element in response.css(self.GAZETTES_CSS):
+            url = self.extract_url(element)
+            date = self.extract_date(element)
+            if url and date and date not in dates:
+                dates.add(date)
+                yield Gazette(
+                    date=date,
+                    file_urls=[url],
+                    is_extra_edition=self.is_extra_edition(element),
+                    municipality_id=self.MUNICIPALITY_ID,
+                    power='executive_legislature',
```

Comment:
Looking some gazettes, I saw just executive information. So, I coudn't confirm that.
Did you saw something from the city council (legislature)?

Reply:
Well, we saw some gazettes like this one https://www.prefeiturademossoro.com.br/jom/jom264.pdf with a legislative section. That's why we set it as `executive_legislature`.